lib/internal/util/inspect.js

```diff
@@ -1099,6 +1099,19 @@ function formatRaw(ctx, value, recurseTimes, typedArray) {
   try {
     output = formatter(ctx, value, recurseTimes);
     for (i = 0; i < keys.length; i++) {
+      if (
+        isDataView(value) && (keys[i] === 'byteLength' || keys[i] === 'byteOffset')
```

Comment:
isDataView makes a c++ call and since this runs on every element, I expect that this is going to impact the performance of this function a lot.

Reply:
hmm, is there a better way you can suggest? i could certainly cache the check outside the for loop, at least

Reply:
ok, i've updated the PR so:
1. it uses the existing `isDataView` call whenever possible
2. it only calls it once in the case where it can't use that call, and that value is cached through the entire loop.

good catch!